sdk/identity-v2/azure-identity/src/main/java/com/azure/v2/identity/AzurePowerShellCredential.java

```diff
@@ -68,12 +70,11 @@ public AccessToken getToken(TokenRequestContext request) {
             AccessToken accessToken = devToolslClient.authenticateWithAzurePowerShell(request);
             LoggingUtil.logTokenSuccess(LOGGER, request);
             return accessToken;
-        } catch (Exception ex) {
-            LoggingUtil.logTokenError(LOGGER, request, ex);
-            if (devToolslClient.getClientOptions().isChained()) {
-                throw LOGGER.logThrowableAsError(new CredentialUnavailableException(ex.getMessage(), ex));
-            }
-            throw LOGGER.logThrowableAsError(new CredentialAuthenticationException(ex.getMessage(), ex));
+        } catch (RuntimeException ex) {
```

Comment:
why are we replacing Exception with RuntimeException everywhere ?

Reply:
for clarity and control - there is nothing that throws checked exceptions under it, so there is no need to catch them.